### PR TITLE
Fix cc1101 detection

### PR DESCRIPTION
**Pull request by sidey79**
* **Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added / modified (needed for for bug fixes / features)
- [ ] commandref has been added / updated (needed for bug fixes / features)
- [x] CHANGED has been updated (needed for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Bug fix

* **What is the current behavior?** (You can also link to an open issue here)

Detection of cc1101 command was broken. Internal hasCC1101 was set to wrong value 

* **What is the new behavior (if this is a feature change)?**

Detection of cc1101 command is corrected. 

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

no

* **Other information**:

Bug was introduced via: 4e7a90c5cd3482701bf70112e1c3853ac5e47f17

### CODE REVIEW

File: FHEM/00_SIGNALduino.pm [hunk 1]
```diff
@@ -1201,14 +1201,13 @@ sub SIGNALduino_CheckCmdResp($)
...
 			SIGNALduino_SimpleWrite($hash, "XE"); # Enable receiver
-			#DevIo_SimpleWrite($hash, "XE\n",2);
 			$hash->{logMethod}->($hash, 3, "$name/init: enable receiver (XE)");
 			delete($hash->{initretry});
 			# initialize keepalive
 			$hash->{keepalive}{ok}    = 0;
 			$hash->{keepalive}{retry} = 0;
 			InternalTimer(gettimeofday() + SDUINO_KEEPALIVE_TIMEOUT, "SIGNALduino_KeepAlive", $hash, 0);
-		 	$hash->{hasCC1101} = 1  if ($ver !~ m/cc1101/);
+		 	$hash->{hasCC1101} = 1  if ($ver =~ m/cc1101/);
```
Comment (elektron-bbs) [inline FHEM/00_SIGNALduino.pm]: Bitte nicht auf den kompletten String "cc1101" testen. Der kann unterschiedlich sein:

SIGNALduino - commands.h:
	#ifdef CMP_CC1101
				MSG_PRINT(FPSTR(TXT_CHIP)); MSG_PRINT(FPSTR(TXT_BLANK)); MSG_PRINT(FPSTR(TXT_CC110));
				switch (cc1101::chipVersion()) {
					case 0x03:
						 MSG_PRINT("0"); 
						break;
					case 0x14:
					case 0x04:
						 MSG_PRINT("1");
						break;
					case 0x05:
						 MSG_PRINT("0E");
						break;
					case 0x07:
						 MSG_PRINT("L");
						break;
						break;
					default:
						MSG_PRINT(" unknown");
						break;
				}

**Comment by sidey79** [comment]
#688

File: FHEM/00_SIGNALduino.pm (hunk 1 above)
Comment (sidey79) [reply FHEM/00_SIGNALduino.pm]: Ich denke das müsste passen, da  TXT_CC110 exakt den Wert cc1101 enthält:

https://github.com/RFD-FHEM/SIGNALDuino/blob/614fa70cbb07ef3de98b008020561ee6790f372d/src/_micro-api/libraries/output/src/output.h#L10

Welcher Chip danach, ist für die Bereitstellung der Kommandos hoffentlich nicht wichtig.

Comment (elektron-bbs) [reply FHEM/00_SIGNALduino.pm]: Ja, ich denke du hast Recht. Die Versionsausgabe enthält CC110x ja zweimal:

V 3.3.1-experimental SIGNALduino cc1101 (chip CC1100) - compiled at Sep 22 2019 22:53:27

**Pull request merged**